Event/src/TrigScintCluster.cxx

```diff
@@ -0,0 +1,41 @@
+#include "Event/TrigScintCluster.h"
+
+ClassImp(ldmx::TrigScintCluster)
+
+namespace ldmx {
+
+  TrigScintCluster::TrigScintCluster() { //: EcalCluster() {
```

Comment:
Since you are using the default cluster, there is no need to have an empty constructor in the src file.  Instead, in the header file just do 
    `TrigScintCluster() = default;`